modules/library/render/src/main/java/org/geotools/renderer/lite/gridcoverage2d/GridCoverageReaderHelper.java

```diff
@@ -256,7 +256,7 @@ private boolean isNotEmpty(ReferencedEnvelope envelope) {
     }
 
     private GridCoverage2D cropCoverage(GridCoverage2D coverage, ReferencedEnvelope cropEnvelope) {
-        final ParameterValueGroup param = CROP.getParameters();
+        final ParameterValueGroup param = PROCESSOR.getOperation("CoverageCrop").getParameters();
```

Comment:
Since a user could change the OperationDescriptor related to a GeoTools operation, users have to get the operation from the Processor again. They should not instantiate it directly, since the OperationDescriptor should have changed. Anyway I will be more detailed in the documentation and in comments.